client_ext_test.go

```diff
@@ -186,6 +188,81 @@ func TestGetNotModified(t *testing.T) {
 	assert.Equal(t, http.MethodGet, unaryReq.HTTPMethod())
 }
 
+func TestSpecSchema(t *testing.T) {
+	t.Parallel()
+	mux := http.NewServeMux()
+	mux.Handle(pingv1connect.NewPingServiceHandler(
+		pingServer{},
+		connect.WithInterceptors(&assertSchemaInterceptor{t}),
+	))
+	server := memhttptest.NewServer(t, mux)
+	testcases := []struct {
```

Comment:
I test over all protocols, but the method should be invariant to it. Shall I reduce to only one and a unary/stream test?

Reply:
I've removed the extra testcases.